patientsearch/api.py

```diff
@@ -210,11 +211,21 @@ def external_search(resource_type):
 
     """
     token = validate_auth()
-    external_search_bundle = external_request(token, resource_type, request.args)
+    external_search_bundle = add_identifier_to_resource_type(
+        bundle=external_request(token, resource_type, request.args),
+        resource_type=resource_type,
+        identifier={
+            'system': current_app.config.get('EXTERNAL_FHIR_API'),
```

Comment:
This config will evaluate to [`http://pdmp:8008/v/r2/fhir/` from cosri-environments](https://github.com/uwcirg/cosri-environments/blob/f9c1cf4114dc76429929276f68bf7e5a6a7b2675/dev/freestanding/femr/docker-compose.yaml#L18)

we may want to mirror the [`system` the FHIR facade uses](https://github.com/uwcirg/script-fhir-facade/blob/5ac23fa8af278e67a7d00247ef83e31e2019f8c7/script_facade/client/config.py#L5-L6)
```suggestion
            'system': 'https://github.com/uwcirg/script-fhir-facade/',
```